src/components/modals/DeleteOwnAccountModal.vue

```diff
@@ -72,11 +76,19 @@
         setup() {
             const busy = ref(false);
             const baseModal = ref();
-            let hasError = ref(false);
-            let password = ref("");
-            let passwordFieldType = ref("password");
-            let loginResponseHandler: LoginResponseHandler = new LoginResponseHandler();
-            let valid = computed(() => password.value != "");
+            const hasError = ref(false);
+            const password = ref("");
+            const passwordFieldType = ref("password");
+            const loginResponseHandler: LoginResponseHandler = new LoginResponseHandler();
+            const valid = computed(() => password.value != "");
+            const enrollmentId = ref("");
```

Comment:
should be secret

Reply:
![grafik](https://user-images.githubusercontent.com/63233799/108390967-7baedd00-7211-11eb-9990-c2c4915f6202.png)

Done with https://github.com/upb-uc4/ui-web/pull/856/commits/8be58a330f87609bf560ad4777dda1de8cb362b1